### PR TITLE
Format and improve doc comments

### PR DESCRIPTION
**Pull request by DalekCraft2**
I also fixed stray typos in normal comments and one in `README.md`.

### CODE REVIEW

**Comment by Geokureli** [comment]
Screenshot 2023-03-29 at 4 57 02 PM

Pretty sure I specifically said in discord I'd rather have many smaller PRs. this is gonna take forever to look over and I don't have that time right now

File: flixel/FlxBasic.hx [hunk 1]
```diff
@@ -21,7 +21,7 @@ class FlxBasic implements IFlxDestroyable
 	#end
 
 	/**
-	 * A unique ID starting from 0 and increasing by 1 for each subsequent `FlxBasic` that is created.
+	 * A unique ID starting from `0` and increasing by `1` for each subsequent `FlxBasic` that is created.
```
Comment (Geokureli) [inline flixel/FlxBasic.hx:24]: I don't want numbers wrapped in backticks

File: flixel/FlxBasic.hx [hunk 2]
```diff
@@ -39,7 +39,7 @@ class FlxBasic implements IFlxDestroyable
 	public var visible(default, set):Bool = true;
 
 	/**
-	 * Useful state for many game objects - "dead" (`!alive`) vs `alive`. `kill()` and
+	 * Useful state for many game objects - "dead" (`!alive`) vs. `alive`. `kill()` and
```
Comment (Geokureli) [inline flixel/FlxBasic.hx:42]: this will help no one

File: flixel/FlxBasic.hx [hunk 3]
```diff
@@ -56,8 +56,8 @@ class FlxBasic implements IFlxDestroyable
 
 	/**
 	 * This determines on which `FlxCamera`s this object will be drawn. If it is `null` / has not been
-	 * set, it uses the list of default draw targets, which is controlled via `FlxG.camera.setDefaultDrawTarget`
-	 * as well as the `DefaultDrawTarget` argument of `FlxG.camera.add`.
+	 * set, it uses the list of default draw targets, which is controlled via `FlxG.camera.setDefaultDrawTarget()`
+	 * as well as the `DefaultDrawTarget` argument of `FlxG.camera.add()`.
```
Comment (Geokureli) [inline flixel/FlxBasic.hx:60]: this will help no one

**Comment by Geokureli** [comment]
I looked at one fraction of one percent of your changes and most of what I saw are, not only frivolous, but not something I even want changed. putting a period after "vs" is a regional thing. 

There's no way that these changes are worth my time, nor yours.

**Comment by Geokureli** [comment]
the point is I'm not gonna go through and manually approve/disapprove each type of change because that means I need to go through 7000 lines. I would much rather both of us spend time on something that will have a bigger impact.

Don't go through the codebase and make stylistic changes. Identify actual problems with the codebase and fix them, or look for important features and implement them. put energy into things that will actually move the needle.

I've said it before and I'll say it again. Make and release HaxeFlixel games, focus on that, first. in the process if see an issue, fix that. don't comb the repo and add periods to the end of one line comments, or change perfectly valid grammar just because you don't like hyphens.

**Comment by DalekCraft2** [comment]
If I was to remake this PR without including any of the pointless formatting/preferential changes, would it be acceptable?

**Pull request closed without merging**